src/main/java/net/sf/jabref/logic/util/strings/StringUtil.java

```diff
@@ -646,7 +646,14 @@ public static String expandAuthorInitials(String name) {
                 }
                 for (int j = 1; j < names.length; j++) {
                     if (j == 1) {
-                        sb.append(Author.addDotIfAbbreviation(names[j]));
+                        String[] splittedNames = names[j].split(" ");
+                        if (splittedNames.length > 0) {
+                            sb.append(Author.addDotIfAbbreviation(splittedNames[0]));
+                        }
+                        for (int k = 1; k < splittedNames.length; k++) {
+                            sb.append(' ');
+                            sb.append(Author.addDotIfAbbreviation(splittedNames[k]));
+                        }
```

Comment:
I am not sure if that's needed here, because the whole method `expandAuthorInitials()` doesn't use it all. This might be the case because the string that gets splitted is used to build a new string via `StringBuilder`.